src/objectTranslations/DataImport__c-es.objectTranslation

```diff
@@ -0,0 +1,543 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObjectTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <caseValues>
+        <plural>false</plural>
+        <value><!-- NPSP Data Import --></value>
+    </caseValues>
+    <caseValues>
+        <plural>true</plural>
+        <value><!-- NPSP Data Import --></value>
+    </caseValues>
+    <fields>
+        <label>Cuenta1 Estado de importación</label>
+        <name>Account1ImportStatus__c</name>
+    </fields>
+    <fields>
+        <label>Cuenta1 Importada</label>
+        <name>Account1Imported__c</name>
+        <relationshipLabel>NPSP Data Imports (Cuenta1 Importada)</relationshipLabel>
+    </fields>
+    <fields>
+        <help>Cuenta1.BillingCity</help>
```

Comment:
Great catch!

I have removed the translated help text from fields where the help text was a field name.

There are other translations for fields where I think it might be valuable to include the translated help text, though.  I kept some of the translated help strings in this file.  Can you look it over again and let me know if any of those would affect the data import process?

For reference, the fields where there is translated help text are:
* `ApexJobId__c`
* `Donation_Donor__c`
* `Donation_Record_Type_Name__c`
* `FailureInformation__c`
* `ImportedDate__c`
* `Status__c`